src/utils/plotSpecs/generateMitochondrialFractionScatterplot.js

```diff
@@ -78,11 +78,7 @@ const generateSpec = (config, plotData) => {
           [
             'blue', 'green', 'grey',
           ],
-        domain: {
-          data: 'plotData',
-          field: 'status',
-          sort: true,
-        },
+        domain: ['Alive', 'Dead'],
```

Comment:
This is a minor bugfix unrelated to this ticket. It fixes the mitochondrial scatterplot's colors.